### PR TITLE
LPS-154778 A permission inconsistency makes the portal unusable

### PR DESCRIPTION
**Pull request by manuelRives**
This solution is an extension of the solution applied on the fix LPS-142128.

Whenever is an inconsistency on the permissions of a single layout the portal remains inoperable. Once the inconsistency appears you cannot even log into the portal. As soon as you pass the login screen appears the error message "An unexpected system error occurred" and you can't access any part of the portal.

The problem with this error is that it appears even when you are accessing a different page that the one with the inconsistency. If the inconsistency exists the whole portal is unusable.

This fix only handles the exception that currently is shown on the portal web interface and logging the existing error into the product logs, keeping the portal accessible while the inconsistency is solved.

### CODE REVIEW

**Comment by manuelRives** [comment]
ci:test:sf

**Comment by manuelRives** [comment]
ci:test:acceptance

**Comment by manuelRives** [comment]
ci:test:relevant

**Comment by manuelRives** [comment]
Hi @liferay-core-infra, I've reviewed the failed test and the don't seem to be related with my changes.

File: portal-impl/src/com/liferay/portal/events/ServicePreAction.java [hunk 1]
```diff
@@ -675,14 +675,28 @@ private LayoutComposite _getViewableLayoutComposite(
...
+				}
+			}
+			catch (Exception exception) {
+				if (_log.isWarnEnabled()) {
+					_log.warn(
+						"Error processing layout " + layout.getLayoutId());
 				}
 
-				accessibleLayouts.add(curLayout);
+				if (_log.isDebugEnabled()) {
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/events/ServicePreAction.java:697]: @manuelRives It is weird you have both Debug and Warn log for same exception, please remove the debug one here.

**Comment by tinatian** [comment]
@manuelRives Please see my comment and resend.

Thanks.
Tina.

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**